apis/r/README.md

```diff
@@ -20,7 +20,7 @@ $ install.packages('tiledbsoma', repos = c('https://tiledb-inc.r-universe.dev',
 
 * Source installation requires the [`tiledb` R package](https://github.com/TileDB-Inc/TileDB-R) (which in turn depends on the [`tiledb` Core library](https://github.com/TileDB-Inc/TileDB)).
 * In general, source installation of TileDB and its packages may require `cmake` and `git` to be installed; these are common tools each operating system provides readily.
-* Other R package dependencies, are listed in the [DESCRIPTION](https://github.com/single-cell-data/TileDB-SOMA/blob/main/apis/r/DESCRIPTION) file and can be installed via _e.g_
+* Other R package dependencies, are listed in the [DESCRIPTION](https://github.com/single-cell-data/TileDB-SOMA/blob/main/apis/r/DESCRIPTION) file and can be installed via _e.g._
   `remotes::install_deps(".")`. In order build vignettes, `knitr` and `rmarkdown` are required as is `testthat` for testing.
```

Comment:
`install_deps()` needs the second argument to install the data package so please change to

     install.deps(".", TRUE)